app/api/v2/models/users_model.py

```diff
@@ -32,7 +32,16 @@ def save(self,firstname,lastname,email,password,phoneNumber,passportUrl,role):
              VALUES('{}','{}','{}','{}','{}','{}','{}')\
              RETURNING firstname,lastname,email,password,phoneNumber,passportUrl,role'''\
             .format(firstname,lastname,email,password,phoneNumber,passportUrl,role))
-		create = self.curr.fetchone()
+		user = self.curr.fetchone()
 		self.conn.commit()
 		self.curr.close()
-		return create
\ No newline at end of file
+		return user
```

Comment:
In this file, the `__init__.py` and the `save` have alot of arguments. Please use `*args` to make your code complexity.